shootout/shootout.py

```diff
@@ -142,32 +152,132 @@ async def soset_bet(self, ctx, amount: int):
         await ctx.tick()
         return
 
-    @soset.command(name="message")
-    async def soset_message(self, ctx, message: str):
-        """Sets the message that the bot listens for during shootout games. Only affects new games!"""
-        await self.config.guild(ctx.guild).message.set(message)
+    @soset.command(name="draw")
+    async def soset_draw(self, ctx, *, draws: str):
+        """Sets the message that the bot listens for during shootout games.
+        One will randomly be selected per game.
+        """
+
+        await self.config.guild(ctx.guild).Messages.set(shlex.split(draws))
         await ctx.tick()
         return
 
-    @soset.command(name="wait")
+    @soset.command(name="victory")
+    async def soset_victory(self, ctx, *, victories: str):
```

Comment:
Few things. First, reference the comment above for the soset_draw function about listing currently set victories.

The help text for this function doesn't mention you have to include {} brackets in your victory messages so the proper stuff gets set. And as you mention here will cause errors, which isn't good and not user friendly. What you should do is look at the welcome cog on how it conditionally formats messages. You should see use named parameters for format method of strings. 

Then, in your help text here you can mention: `Use {winner} to specify the winner, {amount} to specify the amount` etc. In your code to format the victory message you would do something like `VICTORY_MSG.format(winner=m.author.mention, amount=pot)`. 

https://www.dummies.com/programming/python/use-named-arguments-in-format-strings/

This way, no errors will happen if people forget to add the {winner} or other things in their victory messages.
Maybe even put in a check if you get victory messages with one word to give them a warning.

Finally, a suggestion I have to improving the ease of use is to make separate functions to add/delete victories, same with draw messages above. If you don't do this, then if I want to add a single victory message, I would first have to copy all the previous ones, then add mine, then run the command, which isn't very user friendly. I know its easier to do it this way in code but ultimately it'll be better off setting it up like that.

Reply:
* Implemented victory add, victory list, and victory rem command that functions similar to paths in CogManagerUI.
* Implemented user friendly argument format in victory messages.
* Adds a warning if not all arguments are present in the victory message when adding.